onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -3572,17 +3572,87 @@ def aten_max_pool1d_with_indices(
     raise NotImplementedError()
 
 
+@torch_op("aten::max_pool2d", trace_only=True)
 def aten_max_pool2d(
     self: TensorType,
     kernel_size: Sequence[int],
-    stride: Optional[Sequence[int]] = None,
-    padding: Sequence[int] = (0, 0),
+    stride: Sequence[int] = None,
+    padding: Sequence[int] = 0,
     dilation: Sequence[int] = (1, 1),
```

Comment:
@justinchuby Found a conflict. In this case, we should annotate dilation as Sequnece[int], but aten signature has default dilation=1. Do we annotate it as `dilation: sequence[int] = (1,)` ?

```yaml
max_pool2d(Tensor self, int[2] kernel_size, int[2] stride=[], int[2] padding=0, int[2] dilation=1, bool ceil_mode=False) -> Tensor
```

Reply:
No matter what it is, but should be align with dialtion's format.